packages/widgets/src/Pagination/PageSizeSelect.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import propTypes from '@dhis2/prop-types'
+import i18n from '@dhis2/d2-i18n'
+import { SingleSelect, SingleSelectOption } from '@dhis2/ui-core'
+import { spacers } from '@dhis2/ui-constants'
+import { sectionStyles, selectStyles } from './Pagination.styles'
+
+const PageSizeSelect = ({ dataTest, pageSize, pageSizes, onChange }) => (
+    <div className="section" data-test={`${dataTest}-pagesize`}>
```

Comment:
See `Pagination.js`'s comment about section

Reply:
There are additional styles here which I think should have their own css rule inside this file that's provided to the `PaginationSection` component I'm proposing, that would add meaning to the styles through the rule's name